scripts/is/intg/run-int-test.sh

```
@@ -0,0 +1,289 @@
+log_info "Exporting JDK"
+set_jdk ${JDK_TYPE}
+
+pwd
+db_file=$(jq -r '.jdbc[] | select ( .name == '\"${DB_TYPE}\"') | .file_name' ${INFRA_JSON})
+wget -q https://integration-testgrid-resources.s3.amazonaws.com/lib/jdbc/${db_file}.jar  -P $TESTGRID_DIR/${PRODUCT_PACK_NAME}/repository/components/lib
+
+
+sed -i "s|DB_HOST|${CF_DB_HOST}|g" ${INFRA_JSON}
+sed -i "s|DB_USERNAME|${CF_DB_USERNAME}|g" ${INFRA_JSON}
+sed -i "s|DB_PASSWORD|${CF_DB_PASSWORD}|g" ${INFRA_JSON}
+sed -i "s|DB_NAME|${DB_NAME}|g" ${INFRA_JSON}
```

Shellcheck (0.11.0)

[info] Possible misspelling: DB_NAME may not be assigned. Did you mean db_name?

(SC2153)